apps/web/src/components/search/index.js

```diff
@@ -20,16 +20,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import * as Icon from "../icons";
 import "./search.css";
 import Field from "../field";
+import { useStore } from "../../stores/theme-store";
 
 function SearchBox(props) {
+  const { theme } = useStore();
+  const autoCompleteTextStyle = {
+    "input:-webkit-autofill": {
+      WebkitTextFillColor: "rgba(231, 225, 225, 0.842) !important",
+      caretColor: "white",
+      fontSize: "inherit !important"
+    }
+  };
```

Comment:
Either move this to a global constant or inline it as well.

Reply:
No It isn't necessary I added it to avoid any conflict we can remove it no problem